package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -328,12 +332,12 @@ def __init__(self, u, epsilon=1, **kwargs):
             Parameters to be passed for the initialization of a
             :class:`DistanceMatrix`.
         """
-        if isinstance(u, Universe):
+        if isinstance(u, AtomGroup) or isinstance(u, Universe):
```

Comment:
Quick question here - what happens if you pass an UpdatingAtomGroup, does the code still work?

Reply:
yes, it works fine. PFA example code snippet I tried on
[example.zip](https://github.com/MDAnalysis/mdanalysis/files/8323412/example.zip)
.

Reply:
Could you add in a test that uses an UpdatingAtomGroup?

Reply:
sure, will add a separate test for updating atomgroups.

Reply:
Hi, I'm not very sure about what kind of test do we want here. Do I make a test out of the example.zip that I attached before and compare the DistMatrix constructed by updating atom group with just universe and selection string of the atom group?

Reply:
@IAlibay I did the above for updating AG checks. Please let me know if it is fine.